e2e/wdio/.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "plugins": [
+    "wdio"
+  ],
+  "extends": "plugin:wdio/recommended",
+  "globals": {
+    "$": true,
+    "$$": true,
```

Comment:
What are $ and $$ referring to here? jQuery? Something else? Are we adding these globals or are using something already present in the Sitecore UI?